### PR TITLE
Support for original PS position option in TDCosmo class

### PR DESCRIPTION
**Pull request by aymgal**
Following the recent new option for getting image positions from PointSource class, when using additional image prediction, this PR adds support for this option in cosmographic fermat potential and time delays methods.

### CODE REVIEW

File: lenstronomy/Analysis/td_cosmography.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def __init__(self, z_lens, z_source, kwargs_model, cosmo_fiducial=None, lens_mod
                                             kwargs_seeing=kwargs_seeing, kwargs_aperture=kwargs_aperture,
                                             anisotropy_model=anisotropy_model, multi_observations=multi_observations)
 
-    def time_delays(self, kwargs_lens, kwargs_ps, kappa_ext=0):
+    def time_delays(self, kwargs_lens, kwargs_ps, kappa_ext=0, original_ps_position=False):
```
Comment (sibirrer) [inline lenstronomy/Analysis/td_cosmography.py:56]: can you add the doc string? You effectively can copy-past it from the routine you are calling afterwards,

File: lenstronomy/Analysis/td_cosmography.py [hunk 2]
```diff
@@ -62,18 +62,18 @@ def time_delays(self, kwargs_lens, kwargs_ps, kappa_ext=0):
         :param kappa_ext: external convergence (optional)
         :return: time delays at image positions for the fixed cosmology
         """
-        fermat_pot = self.fermat_potential(kwargs_lens, kwargs_ps)
+        fermat_pot = self.fermat_potential(kwargs_lens, kwargs_ps, original_ps_position=original_ps_position)
         time_delay = self._lens_cosmo.time_delay_units(fermat_pot, kappa_ext)
         return time_delay
 
-    def fermat_potential(self, kwargs_lens, kwargs_ps):
+    def fermat_potential(self, kwargs_lens, kwargs_ps, original_ps_position=False):
```
Comment (sibirrer) [inline lenstronomy/Analysis/td_cosmography.py:69]: doc string too

Comment (aymgal) [reply lenstronomy/Analysis/td_cosmography.py:72]: Done

File: lenstronomy/Analysis/td_cosmography.py (hunk 1 above)
Comment (aymgal) [reply lenstronomy/Analysis/td_cosmography.py:56]: Done

**Pull request merged**